### PR TITLE
Pvt endpoint(new version) and updated regions.

### PR DESCRIPTION
**Pull request by SuperMohit**
*Description of changes:*

1. Updated the New version of the private endpoint which takes SubnetIds as an array instead of a string.
2. Updated MongoDB versions and deployment region

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by SuperMohit** [comment]
@vsnyc I have updated the MDB version and regions. We have also updated the PrivateEndpoint schema so quickstart also requires a refresh. I have added executed the taskcat for Private endpoint.
https://quickstart-pi.s3.amazonaws.com/quickstart-temp/taskcat_outputs.zip

File: templates/activate-mongodb-atlas-resources.template.yaml [hunk 1]
```diff
@@ -24,8 +24,10 @@ Parameters:
     - "ap-east-2"
     - "ap-southeast-1"
     - "ap-southeast-2"
+    - "ap-southeast-3"
```
Comment (Zuhairahmed) [inline templates/activate-mongodb-atlas-resources.template.yaml:27]: regions look good

File: templates/mongodb-atlas-peering-existingvpc.template.yaml [hunk 2]
```diff
@@ -147,10 +136,9 @@ Parameters:
     Type: String
     Default: "5.0"
     AllowedValues:
-      - "4.0"
-      - "4.2"
       - "4.4"
       - "5.0"
```
Comment (Zuhairahmed) [inline templates/mongodb-atlas-peering-existingvpc.template.yaml:140]: MongoDB Versions look good

File: templates/mongodb-atlas-main.template.yaml [hunk 3]
```diff
@@ -88,27 +88,14 @@ Parameters:
     - "M20"
     - "M30"
     - "M40"
-    - "R40"
```
Comment (Zuhairahmed) [inline templates/mongodb-atlas-main.template.yaml:91]: i made an error, double checked with atlas go-client and indeed most of these are supported MongoDB Cluster Tiers for AWS. can you update to use this list instead: https://github.com/mongodb/go-client-mongodb-atlas/blob/master/mongodbatlas/clusters.go#L250

also please add "R700" + delete "M400" as well as "M700" as these two 2 are not supported.

**Review by Zuhairahmed — CHANGES_REQUESTED** [review]
regions + mongoDB versions look good, but few updates on cluster tiers

File: templates/mongodb-atlas-main.template.yaml (hunk 3 above)
Comment (SuperMohit) [reply templates/mongodb-atlas-main.template.yaml:91]: We have M400 and M700 on UI. Any idea which API UI is using?
Screenshot 2023-02-01 at 12 11 05 PM

Comment (SuperMohit) [reply templates/mongodb-atlas-main.template.yaml]: updated as per the suggestion:
https://github.com/aws-quickstart/quickstart-mongodb-atlas/pull/30/commits/f0813265c133ccf7c2a459159358541bc2e6b768

**Review by Zuhairahmed — APPROVED** [review]
LGTM

**Review by vsnyc — APPROVED** [review]

**Pull request merged**